src/shared/assets/sass/blocks/_heavy-line.scss

```diff
@@ -0,0 +1,4 @@
+.heavy-line {
```

Comment:
I think this would follow our conventions if it extended and was named like the corresponding GOV.uk component, e.g.:
```
.section-break {
  @extend .gov-uk-section-break;
}
.section-break__heavy {
  @extend .govuk-section-break--visible;
  border-bottom-height: 2px;
}
```